test/http/server.test.js

```diff
@@ -0,0 +1,201 @@
+import { beforeAll, jest, it, expect, afterAll, describe } from "@jest/globals";
+import { shutdown, start } from "../../src/http/server";
+import { DatabaseConnections } from "../../src/infra/database-connections";
+import { mockMongoDriver } from "../util/mongoDbDriverMock";
+import request from "supertest";
+import { getId } from "../../src/infra/idManager";
+
+let app;
+beforeAll(async () => {
+	DatabaseConnections.connect = jest.fn(({ collection }) => {
+		return mockMongoDriver(collection);
+	});
+	jest.spyOn(DatabaseConnections, "status", "get").mockReturnValue("Connected");
+	app = await start(3000, { connectionString: "", databaseName: "" });
+
+	// This line disables logs from the application to avoid pollution during testing
+	console.log = () => {};
+});
+afterAll(async () => {
+	await shutdown();
+});
+
+describe("Follow-up Server", () => {
+	it("Should pass the health check", async () => {
```

Comment:
O que quer dizer should pass? O que o heath check precisa responder pra `should pass`?